### ISSUE TITLE
[BUG] Kirby-dropdown with expand="block" inside fixed with div truncates long texts but has no room for kirby-icon

### ISSUE THREAD

**Issue by ETBoye** [kirbydesign/designsystem#3286]
- [ X ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
A kirby-dropdown with expand="block" inside a fixed-width container will truncate long texts to fit inside the select, but the kirby icon will be pushed outside the box.

## Describe how to reproduce the bug
The bug is demonstrated in [this stackblitz](https://stackblitz.com/edit/angular-x2bjsu?file=src%2Fkirby-demo%2Fkirby-demo.component.html,src%2Fkirby-demo%2Fkirby-demo.component.css).  

Select the dropdown item with the long text.
Inspect the dropdown to show the kirby-icon. It is outside the parent div.

## Which Kirby version was used?
^8.1.4 

## What was the expected behavior?
The kirby icon should fit inside the "select" part of the kirby dropdown

## Please complete the following information:
- OS: Windows
- Browser: firefox
- Browser version: 119.0.1

## Are there any additional context?
I spoke to Rasmus about the bug.

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
